truffle.js

```diff
@@ -16,9 +16,9 @@ module.exports = {
   networks: {
     live: {
       network_id: 1 // Ethereum public network
+      host: "10.135.82.20",
```

Comment:
so we are going to deploy contract from droplet?

Reply:
yes